CMakeLists.txt

```diff
@@ -86,8 +86,14 @@ if(EXISTS "${CMAKE_SOURCE_DIR}/plugins")
 endif()
 
 #-------------------------------------------------------------------------------
+if(NOT PACKAGE_MODE)
```

Comment:
And if you're willing to do it correctly (Easier to read.)
I think it would be better to pu the new section in the bottom.
if (package_mode)
the old cole
else()
your new code
endif()